src/prefect/backend/flow_run.py

```diff
@@ -660,7 +687,7 @@ def _from_flow_run_data(
     def from_flow_run_id(
         cls,
         flow_run_id: str,
-        load_static_tasks: bool = True,
+        load_static_tasks: bool = False,
```

Comment:
No longer load static tasks by default; it's expensive and often we are just inspecting flow run state